src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -0,0 +1,358 @@
+# Command-line arguments
+
+Here's the list of arguments you can pass to `rustdoc`:
```

Comment:
Nope:

```bash
$cargo doc --passes strip-priv-imports
error: Unknown flag: '--passes
…
```

I think it was once possible to do `cargo doc -- args…` but with latest stable:

```bash
$ cargo doc -- --passes strip-priv-imports
error: Invalid arguments.
```